include/Reader/reader.h

```diff
@@ -3279,6 +3292,102 @@ class ReaderBase {
 
   static bool rdrIsMethodVirtual(uint32_t MethodAttribs);
 
+  /// \brief Return MethodName for non helper and non native Methods.
+  ///
+  /// \param method  Method handle for the target Method.
+  /// \param classNamePtr out param, Module name.
+  /// \param JitInfo The jit interface for this method
+  /// \returns MethodName.
+  // TODO sandreenko : change cases to LLILC rules or transfer to coreclr.
+  const char *GetMethodName(CORINFO_METHOD_HANDLE method,
+                            const char **classNamePtr, ICorJitInfo *JitInfo);
+
+
+  /// \brief Return true if we process this Class with SIMD intrinsics.
+  ///
+  /// \param Class The class handle for the call target method's class.
+  /// \returns IRNode * LLVM::int(1 or 0).
```

Comment:
Instead of leaking the LLVM types into this header file, please change the description to something like

```
A value representing true or false (dependent on whether or not vector HW acceleration is enabled)
```